src/libsyntax_pos/symbol.rs

```diff
@@ -804,9 +804,9 @@ impl Ident {
         Ident::new(self.name, self.span.modern_and_legacy())
     }
 
-    /// Convert the name to a `LocalInternedString`. This is a slowish
-    /// operation because it requires locking the symbol interner.
-    pub fn as_str(self) -> LocalInternedString {
+    /// Convert the name to a `SymbolStr`. This is a slowish operation because
+    /// it requires locking the symbol interner.
```

Comment:
cc @Manishearth could `elsa` have a `FrozenIndexSet` that is also thread-safe without a lock?
In this case, just a `Sync` `FrozenVec` might be enough.